slasher/db/kv/spanner.go

```diff
@@ -0,0 +1,306 @@
+package kv
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	"github.com/boltdb/bolt"
+	"github.com/pkg/errors"
+	"github.com/prysmaticlabs/prysm/shared/bytesutil"
+	"github.com/prysmaticlabs/prysm/slasher/detection/attestations/types"
+	log "github.com/sirupsen/logrus"
+	"go.opencensus.io/trace"
+)
+
+var highestEpoch uint64
```

Comment:
This is just maintained in memory?

Reply:
i thought of using it later to choose which epoch to cache